### ISSUE TITLE
Property button hit target is too small

### ISSUE THREAD

**Issue by ermau** [xamarin/Xamarin.PropertyEditing#361]
The hit target should be closer to the height of a standard row, not 1px bigger than that tiny circle.

**Comment by CartBlanche** [comment]
@ermau Fair enough. I think we'll need some of @vancura's time to give us bigger icons and what he thinks from a design perspective would work with our aesthetic.

**Comment by ermau** [comment]
We may want to make the icon bigger, but increasing the hit area does not require that.

**Comment by vancura** [comment]
Yeah, please just increase the hit area. Optically it's good as it is. I can, of course, enlarge bounding box of the icon, but if there's a way to change the hit area with code, please do that instead. I'd like to keep the icon sizes standard.

**Comment by CartBlanche** [comment]
Hit Area increased to 20 pixels. icon the some. Part of this PR - https://github.com/xamarin/Xamarin.PropertyEditing/pull/363
